src/components/Checkbox/Checkbox.module.scss

```diff
@@ -0,0 +1,117 @@
+@import '../../styles/colors';
```

Comment:
Need to run prettier to tidy up the spacing, remove usage of &, tidy up some typo's and ensure we're using Pascal case for each classname (.HasOnlyLink)

Reply:
Updated the styles name and ran the prettier to tidy up the spacing